CHANGELOG.md

```diff
@@ -27,6 +27,9 @@ Changes can also be flagged with a GitHub label for tracking purposes. The URL o
 ### Fixed
 - Updated relationships for Comments, Attachments and PrivacyRequests to remove overlap sqlalchemy error. [#5929](https://github.com/ethyca/fides/pull/5929)
 
+### Removed
+- Removed datasetClassificationUpdates flag and related functionality from admin UI. [#HA-427](https://github.com/ethyca/fides/pull/HA-427)
```

Comment:
What does "related functionality from admin UI" mean here? We didn't remove functionality did we, just the ability to swap the flag on/off?